openadmet/models/architecture/mtenn.py

```diff
@@ -106,4 +131,4 @@ def make_new(self) -> "MTENNSchNetModel":
         """
         Copy parameters to a new model instance without copying the estimator
         """
-        return self.__class__(**self.mod_params, **self.dict(exclude={"estimator"}))
+        return self.__class__(**self.dict(exclude={"estimator"}))
```

Comment:
`dict` method is deprectated, use `model_dump` (it's functionally equivalent). I.e.:

```suggestion
        return self.__class__(**self.model_dump(exclude={"estimator"}))
```